xoinvader/animation.py

```diff
@@ -0,0 +1,103 @@
+"""Animation.
+
+Animation is set of keyframes.
+Value of selected attribute changes in time.
+
+Keyframe:
+  (time, value)
+
+Objects have animation manager which manages animation graph and switching."""
+
+
+from operator import itemgetter
+
+from xoinvader.utils import Timer
+
+
+# TODO: Implement animation graph and etc
+class AnimationManager(object):
+    """Manage list of object animation."""
+
+    def __init__(self):
+        self._animations = {}
+        self._current = None
+
+    def bind(self, name, obj):
+        """Transfer animation to other object."""
+        self._animations[name].bind(obj)
+
+    def add(self, name, *args, **kwargs):
+        """Add new animation, pass args to Animation class.
+
+        :param str name: animation name
+        """
+
+        self._animations[name] = Animation(name, *args, **kwargs)
+        if len(self._animations) == 1:
+            self._current = name
```

Comment:
mapping {name -> animation object}. Switching of current animation will be implemented through changing self._current. Object that holds AnimationManager knows about animation name that it creates via manager, not about <animation.Animation object at 0x...> =)